airflow/www_rbac/templates/airflow/editdag.html

```diff
@@ -359,24 +429,27 @@ <h4 class="modal-title">Replace placeholder values in codebrick.</h4>
             '<label for="' + param + '">' + param + '</label>' +
             '<input type="text" class="form-control" name="' + param + '">' +
             '</div>'
-        } else {
+        }
+        else {
           param = param.replace('{{"{%"}}', '').replace('{{"%}"}}', '')
           temp = '<div class="form-group">' +
             '<label for="' + param + '">' + param + '<span style="color:red;">* </span> </label>' +
             '<input type="text" class="form-control" name="' + param + '" required>' +
             '</div>'
         }
-        $('#insertSnippetBody').append(temp)
+        $('#insertSnippetBody').append(temp);
       })
-      $('#insertSnippet').modal('show')
-    } else {
+      $('#showSnippetDoc').html(desc);
+      $("#modalDocBtn").text("Show Documentation");
+      $('#insertSnippetBody').show();
+      $('#showSnippetDoc').hide();
+      $('#insertSnippet').modal('show');
+    }
+
+    else {
       // adding a new line at the end of snippet.
       replaceTextInEditor(text)
     }
-
-    // disable the button when snippet is inserted. The button will be enabled
-    // after the codebrick div is toggled again.
-    $(btn).addClass('disabled');
```

Comment:
Don't remove this ...?

Reply:
Was causing this behavior:
- if you choose to abandon an attempt to edit code/insert snippet and close the modal, the button is unclickable until you toggle codebricks again. 
Found it problematic as if I choose to abandon an edit attempt and decide to insert the codeblock I can't do it without toggling the codebrick again.

Reply:
shobhit wanted this to happen. Anyway we can comment this for now, don't remove this

Reply:
Okay, will comment it. Editing.